ui/admin/tests/acceptance/host-catalogs/read-test.js

```diff
@@ -71,14 +74,33 @@ module('Acceptance | host-catalogs | read', function (hooks) {
       instances.hostCatalog.authorized_actions.filter(
         (item) => item !== 'read'
       );
-    await visit(urls.hostCatalog);
-    assert.notOk(find('main tbody .rose-table-header-cell:nth-child(1) a'));
+
+    await click(`[href="${urls.hostCatalogs}"]`);
+
+    assert.dom(`[href="${urls.hostCatalog}"]`).doesNotExist();
```

Comment:
We have a test that we can't navigate to a `host catalog` inside `host catalogs` but I think we should also have a test we can't navigate to the `host-catalogs` without proper permissions as well

Reply:
Maybe I'm misunderstanding the suggestion but since host-catalogs is a route and not a model I can't set permissions.

Reply:
I think there might be a little confusion regarding the permissions. `host-catalogs` permission would be the `list` which doesn't live on the individual model, it lives one layer above in the scope and you'd have to mock `authorized_collection_actions` instead of `authorized_actions`. This is true for any of the collections. 

I'm suggesting we test this permission check [here](https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/templates/scopes/scope.hbs#L108).

You can see we mock the permission [here](https://github.com/hashicorp/boundary-ui/blob/main/addons/api/mirage/factories/scope.js#L27)

Reply:
@ZedLi are you suggesting a test that verifies if the user can (or cannot) navigate to the `urls.hostCatalogs` route?

Reply:
I just needed to wait one more second for your comment to come through 😆 
@lisbet-alvarez here is an [example](https://github.com/hashicorp/boundary-ui/blob/6f8afbb7af5927e45a45756150951fc1c63272ab/ui/admin/tests/acceptance/sessions-test.js#L89-L100) of what @ZedLi is suggesting.

Great suggestion Zed!

Reply:
Ahh okay makes sense! Thxx